src/app/pages/dashboard/widgets/system/widget-os-version/widget-os-version.component.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  ChangeDetectionStrategy, Component, input,
+} from '@angular/core';
+import { WidgetResourcesService } from 'app/pages/dashboard/services/widget-resources.service';
+import { WidgetComponent } from 'app/pages/dashboard/types/widget-component.interface';
+import {
+  SlotSize,
+} from 'app/pages/dashboard/types/widget.interface';
+import { osVersionWidget } from 'app/pages/dashboard/widgets/system/widget-os-version/widget-os-version.definition';
+
+@Component({
+  selector: 'ix-widget-os-version',
+  templateUrl: './widget-os-version.component.html',
+  styleUrls: ['./widget-os-version.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class WidgetOsVersionComponent implements WidgetComponent {
+  size = input.required<SlotSize>();
+  readonly name = osVersionWidget.name;
+
+  systemInfo$ = this.resources.systemInfo$;
```

Comment:
I recently added `getSystemVersion` helper function to include release name into version. We may want to show the version consistently.

Reply:
Feel free to rename it to a common name